Source/Events/PartitionId.cs

```diff
@@ -9,25 +9,25 @@ namespace Dolittle.SDK.Events
     /// <summary>
     /// Represents the concept of a unique identifier for a partition.
     /// </summary>
-    public class PartitionId : ConceptAs<Guid>
+    public class PartitionId : ConceptAs<string>
     {
         /// <summary>
         /// The <see cref="PartitionId"/> that refers to no partition.
         /// </summary>
-        public static readonly PartitionId Unspecified = Guid.Empty;
+        public static readonly PartitionId Unspecified = string.Empty;
 
         /// <summary>
         /// Implicitly converts from a <see cref="Guid"/> to an <see cref="PartitionId"/>.
         /// </summary>
         /// <param name="partitionId">The <see cref="Guid"/> representation.</param>
         /// <returns>The converted <see cref="PartitionId"/>.</returns>
-        public static implicit operator PartitionId(Guid partitionId) => new PartitionId { Value = partitionId };
+        public static implicit operator PartitionId(Guid partitionId) => new PartitionId { Value = partitionId.ToString() };
```

Comment:
In JavaScript I removed all the Guid parameters from the `from` static method on the partition and event source concepts, to not cause confusion and to break the API so that we're forced to give strings rather that Guids. I think that is what we want, what do you think?

Reply:
Let's leave this for now until we get the results back from the poll.